vllm/forward_context.py

```diff
@@ -238,6 +238,6 @@ def set_forward_context(attn_metadata: Any,
                 if forward_stats:
                     logger.info(("Batchsize forward time stats "
                                  "(batchsize, count, median_time(ms)): %s"),
-                              forward_stats)
+                                forward_stats)
 
-        _forward_context = prev_context
\ No newline at end of file
+        _forward_context = prev_context
```

Comment:
remove the change here as we didn't change this file

Reply:
That's from pre-commit. I see the same failure from PR1369 so updated it in this PR.
Do you still want to remove it? We will still need to fix it from PR1369 then.